daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/SAXInfosetInputter.scala

```diff
@@ -207,7 +207,8 @@ class SAXInfosetInputter(
       }
     } catch {
       case e: Exception => {
-        batchedInfosetEvents(currentIndex).causeError = One(new DaffodilUnhandledSAXException(e.getMessage, e))
+        batchedInfosetEvents(currentIndex).causeError = One(new DaffodilUnhandledSAXException(e))
+
```

Comment:
Line 210 isn't covered by any test, which causes the new constructor to be uncovered too.  

Does anyone have any idea how to write a test that would cause an exception to escape from dp.unparse at line 202 without being wrapped in an unparseResult.isError?  

@mphyo21, if you want to practice your IDE navigation skills, please use your IDE's Go To functionality to find out which code implements the dp.unparse call at line 202 and inform us so we can check if there is any way a test could make an exception escape from that code.  If the IDE can't tell you which code implements dp.unparse, you can practice your IDE debugging skills instead - set a breakpoint at line 202, run a test from your IDE, step into the dp.unparse call, and report back to us which code implements that call.